### DISCUSSION TITLE
Are there disadvantages to using a created "Cash wallet" Asset account compared to the "(cash)" account?

### DISCUSSION THREAD

**Discussion by kylxbn** [firefly-iii/firefly-iii#6534] [Firefly III] [answered]
I did not read the manual thoroughly before using Firefly III, and I created a "Cash wallet" asset account called "Cash wallet" to track the cash I have in my wallet, and a "Shared asset" account called "House cash" to track the money shared between all members of the house. All instead of using a "(cash)" account (the one that seems to be created when you leave to/from account fields empty).

I like to track where the money goes even after I have them as cash on my wallet (so I make deposit transactions from "Cash wallet" to various expense accounts such as malls and stores), and also have the cash in my wallet included in the computation of net worth / total asset.

What would be the benefits if I were to use a "(cash)" account instead? Would it be better if I were to use a "(cash)" account?

Awesome software, thank you very much for creating this!

**Comment by JC5** [comment, marked as answer]
Thanks for the compliments!

Personally, I never track cash so I created the "(cash)" account specifically for this purpose. Money spent on "(cash)" is basically an blank expense. I always lose change, or get short-changed, or something, so my cash wallet in Firefly III and cash in my hand never match. It's easier for me.

Having said that, people wanted to track their cash specifically through an asset account, so there you go. It's up to you!

**Reply by kylxbn** [reply]
Wow, that is an extremely quick answer! I can't believe you're developing and maintaining such a useful and feature-packed software so actively. I would honestly support the project but... I guess you can imagine that I'm not doing too well financially which is why I want to use this software 😅

Thank you very much for the quick response! I'll stick to using cash wallet accounts, then!

**Reply by JC5** [reply]
Haha that's quite alright. Glad the answer helps!
